src/rgw/rgw_rest_swift.cc

```diff
@@ -914,6 +914,109 @@ void RGWOptionsCORS_ObjStore_SWIFT::send_response()
   end_header(s, NULL);
 }
 
+int RGWBulkDelete_ObjStore_SWIFT::get_data(list<RGWBulkDelete::acct_path_t>& items,
+                                           bool * const is_truncated)
+{
+  const size_t MAX_LINE_SIZE = 2048;
```

Comment:
@yehudasa ```2048``` should be a reasonable value here. Swift has following length limits:
* 256 max. for container name,
* 1024 max. for object name.

radosgw differs here only in the matter of container names. They could have 255 characters max.